src/pages/workspace/WorkspacePeoplePage.js

```diff
@@ -0,0 +1,328 @@
+    policy: PropTypes.shape({
+        /** The policy name */
+        name: PropTypes.string,
+    }),
+
+    /** URL Route params */
+    route: PropTypes.shape({
+        /** Params from the URL path */
+        params: PropTypes.shape({
+            /** policyID passed via route: /workspace/:policyID/people */
+            policyID: PropTypes.string,
+        }),
+    }).isRequired,
+};
+
+const defaultProps = {
+    policy: {
+        name: '',
+    },
+};
+
+class WorkspacePeoplePage extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            selectedEmployees: [],
+            isRemoveMembersConfirmModalVisible: false,
+        };
+
+        this.inviteUser = this.inviteUser.bind(this);
+        this.removeUsers = this.removeUsers.bind(this);
+        this.toggleUser = this.toggleUser.bind(this);
+        this.renderItem = this.renderItem.bind(this);
+        this.renderHeader = this.renderHeader.bind(this);
+        this.askForConfirmationToRemove = this.askForConfirmationToRemove.bind(this);
+        this.hideConfirmModal = this.hideConfirmModal.bind(this);
+    }
+
+    /**
+    * Open the modal to invite a user
+    */
+    inviteUser() {
+        Navigation.navigate(ROUTES.getWorkspaceInviteRoute(this.props.route.params.policyID));
+    }
+
+    /**
+    * Remove selected users from the workspace
+    */
+    removeUsers() {
+        // Remove the admin from the list
+        // eslint-disable-next-line max-len
+        removeMembers(this.state.selectedEmployees.filter(email => email !== this.props.session.email), this.props.route.params.policyID);
+        this.setState(prevState => ({
+            ...prevState,
+            selectedEmployees: [],
+            isRemoveMembersConfirmModalVisible: false,
+        }));
+    }
+
+    /**
+    * Show the modal to confirm removal of the selected members
+    */
+    askForConfirmationToRemove() {
+        this.setState(prevState => ({...prevState, isRemoveMembersConfirmModalVisible: true}));
+    }
+
+    /**
+    * Hide the confirmation modal
+    */
+    hideConfirmModal() {
+        this.setState(prevState => ({...prevState, isRemoveMembersConfirmModalVisible: false}));
+    }
+
+    /**
+    * Add or remove user from the selectedEmployees list
+    *
+    * @param {Sting} login
+    */
+    toggleUser(login) {
+        if (login === 'ALL') {
+            this.setState(prevState => ({
+                ...prevState,
+                selectedEmployees: prevState.selectedEmployees.length !== this.props.policy.employeeList.length
+                    ? this.props.policy.employeeList
+                    : [],
+            }));
+            return;
+        }
+
+        this.setState(prevState => ({
+            ...prevState,
+            selectedEmployees: prevState.selectedEmployees.includes(login)
+                ? prevState.selectedEmployees.filter(item => item !== login)
+                : [...prevState.selectedEmployees, login],
```

Comment:
It might be a little simpler to just do something like 

```js
selectedEmployees: _.uniq([...prevState.selectedEmployees, login]);
```

Reply:
I think this would not work as we need to. `_.uniq` would always keep one instance of `login` in the array so we could not toggle the member `off`.